packages/deployment/app/cloudformation.template.yml

```diff
@@ -1202,13 +1202,125 @@ Resources:
     Properties:
       Cluster:
         Ref: CumulusECSCluster
-      DesiredCount: {{# if this.count}}{{this.count}}{{ else }} 0 {{/if}}
+      DesiredCount: {{# if this.count}}{{this.count}}{{ else }}0{{/if}}
       TaskDefinition:
         Ref: {{@key}}TaskDefinition
       DeploymentConfiguration:
         MaximumPercent: 100
         MinimumHealthyPercent: 0
 
+  {{! If minTasks and maxTasks are not configured for this service, do not configure autoscaling }}
+  {{# if this.minTasks }}{{# if this.maxTasks }}
+
+  {{@key}}ECSServiceScalableTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    Properties:
+      MinCapacity: {{this.minTasks}}
+      MaxCapacity: {{this.maxTasks}}
+      ResourceId:
+        Fn::Sub:
+          - service/${CumulusECSCluster}/${ServiceName}
+          - ServiceName:
+              Fn::GetAtt:
+                - "{{@key}}ECSService"
+                - Name
+      RoleARN: {{../this.iams.scalingRoleArn}}
+      ScalableDimension: ecs:service:DesiredCount
+      ServiceNamespace: ecs
+
+  {{# if this.scaleInAdjustmentPercent }}
+  {{@key}}ECSServiceScalingInPolicy:
+    Type: AWS::ApplicationAutoScaling::ScalingPolicy
+    DependsOn:
+      - {{@key}}ECSServiceScalableTarget
+    Properties:
+      PolicyName: {{../stackName}}{{@key}}ECSServiceScalingInPolicy
+      PolicyType: StepScaling
+      ScalingTargetId: !Ref {{@key}}ECSServiceScalableTarget
+      StepScalingPolicyConfiguration:
+        Cooldown: 60
+        AdjustmentType: PercentChangeInCapacity
+        MinAdjustmentMagnitude: 1
+        StepAdjustments:
+          - MetricIntervalUpperBound: 0
+            ScalingAdjustment: {{this.scaleInAdjustmentPercent}}
+  {{/if}}{{! End of "if this.scaleInAdjustmentPercent" condition }}
+
+  {{# if this.scaleOutAdjustmentPercent }}
```

Comment:
same as above with the `scaleInAdjustmentPercent`

Reply:
See https://github.com/nasa/cumulus/pull/942#issuecomment-484949251